### DISCUSSION TITLE
Programmatically calling logconvert

### DISCUSSION THREAD

**Discussion by snoop911** [hardbyte/python-can#1698] [Q&A]
I'm trying to do the following:
can_logconvert.exe  ascfile.asc  logfile.log

but I want to do in python instead of the windows command line:
import can
can.logconvert     #no attribute logconvert error

How do I use the module from my python script passing in the file handles?

**Comment by grant-allan-ctct** [comment]
Does this [example code](https://github.com/hardbyte/python-can/blob/main/examples/simple_log_converter.py) help?
